netbox/utilities/templatetags/helpers.py

```diff
@@ -13,9 +14,21 @@
 from utilities.forms import TableConfigForm
 from utilities.utils import foreground_color
 
+from extras.plugins import PluginConfig
+
 register = template.Library()
 
 
+def _resolve_namespace(instance):
+    """
+    Get the appropriate namespace for the app based on whether it is a Plugin or base application
+    """
+    app = apps.get_app_config(instance._meta.app_label)
+    if isinstance(app, PluginConfig):
```

Comment:
Perhaps I should check if this is a base app as well.  Because this could allow people to override base app behaviour, thoughts?